examples/vault-ec2-auth/main.tf

```diff
@@ -0,0 +1,210 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A VAULT SERVER CLUSTER AND A CONSUL SERVER CLUSTER IN AWS
+# This is an example of how to launch a vault cluster and then authenticate an instance to the cluster
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.11.7"
+}
+
+resource "aws_instance" "example_auth_to_vault" {
+  ami           = "${var.ami_id}"
+  instance_type = "t2.micro"
+  subnet_id     = "${data.aws_subnet_ids.default.ids[0]}"
+  key_name      = "${var.ssh_key_name}"
+
+  # Uses the same security group as the vault cluster
+  # Plus an additional one that opens the port to our simple web server
```

Comment:
Why does it use the same security group as the vault cluster?

Reply:
I don't remember why I did this! Might have been related to thinking the consul agent would need it? But thinking about it now, I can't think why it would be necessary. I'll remove it